src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/dask-gateway/files/gateway_config.py

```diff
@@ -227,18 +229,23 @@ def base_username_mount(username, uid=1000, gid=100):
     }
 
 
-def worker_profile(options, user):
```

Comment:
I renamed this function since it affects the scheduler as well and not just the worker